### ISSUE TITLE
Add IIIF download

### ISSUE THREAD

**Issue by mejackreed** [geoblacklight/geoblacklight#291]

**Comment by sgbalogh** [comment]
I'm interested in taking a stab at this...

IIIF image download is something of an odd case (compared to the other generated download options) because we might want to collect some additional user input before making the request, like which imgage format for the download, and what size. Does anyone have any thoughts on this –– or on how best to work it into the interface?

**Comment by mejackreed** [comment]
I think that a good start would be just allowing to download the full image. 

```
http://www.example.org/image-service/abcd1234/full/full/0/default.jpg
```

That may remove some complexity of the problem.

It would be great if this could be integrated into the download split button.

**Comment by eliotjordan** [comment]
Closed by #384.
